Dockerfile

```diff
@@ -1,5 +1,11 @@
-FROM starefossen/github-pages:137
+# Set to the version for this archive
+ARG VER=v17.03
 
-COPY . /usr/src/app
+# This image comes from the Dockerfile.onbuild file in the docs-builder branch
+# https://github.com/docker/docker.github.io/blob/docs-builder/Dockerfile.onbuild
```

Comment:
This is now https://github.com/docker/docker.github.io/blob/publish-tools/Dockerfile.builder.onbuild